task1/html/index.css

```diff
@@ -0,0 +1,27 @@
+h1 {
+  text-align: center;
+  color: black;
+  line-height: 100px;
+  margin: 0;
+}
+.container {
+  display: flex;
+  justify-content: center;
+}
+.item {
+  display: flex;
+  justify-content: center;
+  width: 200px;
+  height: 300px;
+  border: 1px solid black;
+  padding: 50px 150px 200px 150px;
```

Comment:
Let's left only top&bottom padding, instead of left/right use flex align-items center

Reply:
and flex-direction should be column